content/applications/finance/fiscal_localizations/austria.rst

```diff
@@ -0,0 +1,137 @@
+(in XML format).
+
+Configuration
+-------------
+
+The following explains how to ensure that all the information that is needed
+for the SAF-T export is available. In case missing information is detected during the export process
+a warning message will appear that lists the missing information.
+
+Company
+~~~~~~~
+
+Under :guilabel:`Settings --> General Settings`, in the **Companies** section, click
+:guilabel:`Update Info` and fill in the following information:
+
+- Fill in the :guilabel:`Address` and :guilabel:`Phone` fields.
+  For the address please enter (at least) :guilabel:`Street`, :guilabel:`City`, :guilabel:`ZIP`
+  and :guilabel:`Country`.
+- Provide your company's tax ID in the :guilabel:`Company ID` field.
+- If you have one, provide your :abbr:`UID-Nummer (Umsatzsteueridentifikationsnummer)`
+  in the :guilabel:`Tax ID` field number, including the country prefix.
+
+You must have at least one **contact person** linked to your company in the **Contacts** app:
+
+- If no **contact person** is linked, create a new one by clicking :guilabel:`New`,
+  set it as :guilabel:`Individual`, and select your company in the :guilabel:`Company name` field.
+- At least one of the :guilabel:`Phone` and :guilabel:`Mobile` fields has to be filled in
+  for the contact person.
+
+Under :guilabel:`Settings --> Accounting`, in the **Austrian localization** section,
+set the following information:
+
+- :guilabel:`ÖNACE-Code`
+- :guilabel:`Profit Assessment Method`
+
+.. _austria/SAFT/chart_of_accounts:
+
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+
+.. seealso::
+   :doc:`../accounting/get_started/chart_of_accounts`
+
+The Austrian SAF-T specification defines a chart of accounts (COA). All accounts that are relevant
+for the SAF-T export have to be annotated with a fitting account from this COA.
+
+The needed mapping information has to be supplied via tags on the accounts.
+Applying the tag `1000` on an account (virtually) maps it to the account with code `1000`
+in the SAF-T COA.
+Similarly any number can be used instead of `1000` to map an account to any account in the SAF-T COA
+(as long as there is an account in the SAF-T COA with that code).
+
+The :guilabel:`Austria - Accounting` module already installs a dedicated tag for each account
+from the SAF-T COA. Furthermore it installs a (suggested) mapping for many of the accounts
+from the default Austrian COA (installed by :guilabel:`Austria - Accounting` module).
+
+To find accounts that are currently unmapped (or mapped to multiple SAF-T accounts)
+you can try to export the SAF-T (:ref:`See below <austria/SAFT/generating_the_document>`).
+A warning message will appear in case of problematic accounts.
+It includes a button through which a list of these problematic accounts can be accessed.
+
+Customer and supplier
+~~~~~~~~~~~~~~~~~~~~~
+
+Fill in the :guilabel:`Address` of each partner that
+appears in your invoices, vendor bills, or payments through the **Contacts** app.
```

Comment:
Fill in the :guilabel:`Address` of each partner that
appears in your invoices, vendor bills, or payments through the :guilabel:`Contacts` app.